recipes/cobra/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Though include the same options here too. Namely, `--single-version-externally-managed --record record.txt`